.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
   - pipenv install --dev -e .
 
 script:
-  - pipenv run py.test --cov=parselglossy tests/ --doctest-module
+  - pipenv run py.test --cov=parselglossy tests/ --doctest-modules
```

Comment:
This option can be entirely nuked as we don't use doctest-style tests at all

Reply:
OK I will modify the PR to remove that flag.